ruleset.md

```diff
@@ -0,0 +1,23 @@
+# Adidas API Guidelines Ruleset
+
+- all JSON fields MUST follow `camelCase`
+- field names MUST be ASCII alphanumeric characters or `_` or `$`
+- collection/array fields MUST have names in plural
+- all requests go through `https` protocol
+- every API operation MUST have at least one `2xx` response
+- `GET` request MUST NOT accept a `body` parameter
+- all success responses MUST be of media type `application/hal+json`
+- all error responses MUST be of media type `application/problem+json`
+- every request SHOULD support `application/json` media type

```

Comment:
What does this mean? That `accept: application/json` is a valid request ?

Reply:
could you elaborate? This is taken straight from the guidelines ;) https://adidas.gitbook.io/api-guidelines/rest-api-guidelines/message/message-formats

Reply:
this means that every endpoint accepting a body should be accepting those of `application/json` type